app/Domains/Users/Policies/UserPolicy.php

```diff
@@ -47,7 +47,7 @@ public function create(User $user): bool
     /**
      * Determine whether the authenticated user can update other users or not.
      *
-     * @param  User $user  The resource entity from the authenticated user.
+     * @param User $user The resource entity from the authenticated user.
```

Comment:
Hey Can u align the User class with the same start as the return type in the docblock.